docs/running-locally.md

```diff
@@ -2,8 +2,8 @@
 
 You have two options:
 
-1. If you're using VSCode, you use the Dev-Container. This takes about 7 minutes. Pre-commit checks take about 4 minutes to run.
-1. Install the requirements on your computer manually. This takes about 1 hour. Pre-commit checks take about 3 minutes to run.
+1. If you're using VSCode, you use the [Dev-Container](/docs/running-locally.md#development-container). This takes about 7 minutes. 
+1. Install the [requirements](/docs/running-locally.md#requirements) on your computer manually. This takes about 1 hour. 
```

Comment:
Hmm this seems long...

Reply:
the one hour? Yeah, I just left that in but I'm not sure about that part.

I did remove the precommit checks lines because I didn't really believe that it would only take that long.